nautobot/docs/development/apps/migration/code-updates.md

```diff
@@ -73,10 +73,46 @@ menu_items = (
 
 ```
 
+### Remove Tag/Tags Filter from FilterSet Definitions
+
+In Nautobot 2.0, you can safely remove `tag = TagFilter(...)` from your filter set definitions as long as your filter sets inherit from `NautobotFilterSet` class and `tags` is added to the `class.Meta.fields`.
+
+For example, before the filter set could look like this:
+
+```py
+
+class AppModelFilterSet(BaseFilterSet):
+
+    name = MultiValueCharFilter(...)
+    number = MultiValueNumberFilter(...)
+    tag = TagFilter(...)
+
+    class Meta:
+        fields = ["name", "number"]
+```
+
+After `tag` filter is removed:
+
+```py
+
+class AppModelFilterSet(NautobotFilterSet):
+
+    name = MultiValueCharFilter(...)
+    number = MultiValueNumberFilter(...)
+
+    class Meta:
+        fields = ["name", "number", "tags"]
+
+```
+
 ## Replace DjangoFilterBackend with NautobotFilterBackend
 
 If your REST API has any `FilterBackend` classes derived from `DjangoFilterBackend`, you should replace `DjangoFilterBackend` with `NautobotFilterBackend`.
 
+## App Model Serializer Inheritance
+
+App Model Serializers for any models that could have a Generic Foreign Key or a Many to Many relationship from a Nautobot Core model **must** inherit from BaseModelSerializer at a minimum so that they have a properly generated `object_type` field. This also applies to the case where your model is a subclass of `ChangeLoggedModel` and you will have a Generic Foreign Key from `ObjectChange`'s `changed_object` field. Otherwise drf-spectacular schema generation will throw an error that looks like this: (drf_spectacular.E001) Schema generation threw exception "Field name `object_type` is not valid for model `YourAppModel`.
```

Comment:
````suggestion
App Model Serializers for any models that could have a Generic Foreign Key or a Many to Many relationship from a Nautobot Core model **must** inherit from BaseModelSerializer at a minimum so that they have a properly generated `object_type` field. This also applies to the case where your model is a subclass of `ChangeLoggedModel` and you will have a Generic Foreign Key from `ObjectChange`'s `changed_object` field. Otherwise drf-spectacular schema generation will throw an error: 

```no-highlight
(drf_spectacular.E001) Schema generation threw exception "Field name `object_type` is not valid for model `YourAppModel`.
```
````